packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -665,8 +665,9 @@ export class InputTimePicker
     }
 
     if (key === "Enter") {
+      event.preventDefault();
```

Comment:
In this example, we always call event.preventDefault for enter regardless of the existence of a form. So we should be consistent with the other comment.

Reply:
Agreed, we should only cancel when the component reacts to the handled event.